CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # CHANGELOG
 
+- Remove `areas` field from BusinessSupportEdition
```

Comment:
I'm not sure I agree with this.  We've often added items to the changelog (under a `unreleased` heading) in PRs.  That way when it comes to release a version, there's no need to figure out what's changed from the git history, the changelog notes are already there.